osu.Game/Online/Rooms/Room.cs

```diff
@@ -111,8 +111,9 @@ private ushort autoStartDuration
         [JsonProperty("current_user_score")]
         public readonly Bindable<PlaylistAggregateScore> UserScore = new Bindable<PlaylistAggregateScore>();
 
+        [Cached]
```

Comment:
this can be reverted if required, i didn't end up using it. or just let it be (was weird to have been missing).

Reply:
I was going to ask.